### ISSUE TITLE
ghostclicks and iScroll

### ISSUE THREAD

**Issue by kosich** [RyanMullins/angular-hammer#42]
For some reason, while using **ng-hammerjs + ng-iscroll** ghostClicks prevention some times fails: tap event handler triggers twice and even scrolls parent ng-iscroll element

some settings I'm using:
ng-hammer `preventGhost : true` -- need it for touch laptops
ng-iscroll `preventDefault: false` -- need it for forms to interact properly (see [this issue](https://github.com/cubiq/iscroll/issues/576#issuecomment-40585882))

so I've come up with a bit different ghost clicks elimination implementation, based on blocking **touch** events for some time after **mouse** events were triggered on the element and *vice versa*.

Please, see [this gist](https://gist.github.com/kosich/23188dd86633b6c2efb7)

replace `preventGhosts` [function  in your](https://github.com/RyanMullins/angular-hammer/blob/master/angular.hammer.js#L447-L521) `angular.hammer.js`with code from the gist ( uncomment the console calls if needed to see whats happening )

**Note**: this has been tested only against chrome's "notebook with touch" emulation and **HAS NOT** been tested on a real device.

So, to sum up: if I'm not alone in running into such an issue and if this code works well on real touch devices I could create a PR.

Thanks.

**Comment by kosich** [comment]
ok, had to close it to make a quickfix. now it seem to work fine ( again, in emulation ). Please, provide some real device experience.

**Comment by RyanMullins** [comment]
Significant changes were merged with #55, please verify that this issue persists with those changes and update accordingly.

**Comment by kosich** [comment]
can't retest it atm, so since noone confirmed having the same issue, Ima close this one. Thx
